Form/Type/CustomFieldType.php

```diff
@@ -191,6 +191,7 @@ private function buildModalFormFields(FormBuilderInterface $builder, array $opti
                 'attr'  => [
                     // Need for JS call of method used for changing values dynamically in the frontend. Check co-form.js
                     'data-toggle-button' => true,
+                    'readonly'           => (bool) $customField->getIsUniqueIdentifier(),
```

Comment:
ReadOnly does not work with radio button, readonly is not property of radio button.

Reply:
Hi, even I had that doubt. I tried this and this is working okay. This works in the backend by  enabling the "disabled attribute. `<label class="btn btn-default disabled btn-no">`

Reply:
@aarohiprasad  toggle is greyed out and disable so it seems good, no issue.